content/docs/06-integrations/00-aws-autoscaler.md

```diff
@@ -14,81 +14,174 @@ import InfoBox from 'shared/components/InfoBox';
+
+</Tabs.TabPane>
+
+<Tabs.TabPane tab="1.24.x" key="1.24.x">
+
+## Prerequisites
 
-**1.22.2**
+* Kubernetes 1.19.x or higher. 
 
 </Tabs.TabPane>
 
 <Tabs.TabPane tab="1.0.x" key="1.0.x">
 
-**1.0.0**
+## Prerequisites
+
+* Kubernetes 1.19.x or higher. 
 
 </Tabs.TabPane>
+
 </Tabs>
 
-## References
+* Permission to create an IAM policy in the AWS account you use with Palette. 
+
+
+* **IAM policy** - The following full Cluster Autoscaler policy should be attached to the cluster's node group. There are two ways to achieve this prerequisite. You can define this as a *customer-managed* policy in the AWS account and use its Amazon Resource Name (ARN) in the cluster profile. Alternatively, you can attach the following IAM policy as an *inline* policy to the node group if you have already deployed your cluster. Refer to the **Usage** section below for more details. <Tooltip>Refer to the [AWS IAM documentation](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_managed-vs-inline.html) for the differences between managed and inline policies.</Tooltip>
+<br />
+
+  ```json
+  {
+    "Version": "2012-10-17",
+    "Statement": [
+      {
+        "Effect": "Allow",
+        "Action": [
+          "autoscaling:DescribeAutoScalingGroups",
+          "autoscaling:DescribeAutoScalingInstances",
+          "autoscaling:DescribeLaunchConfigurations",
+          "autoscaling:DescribeScalingActivities",
+          "autoscaling:DescribeTags",
+          "ec2:DescribeInstanceTypes",
+          "ec2:DescribeLaunchTemplateVersions"
+        ],
+        "Resource": ["*"]
+      },
+      {
+        "Effect": "Allow",
+        "Action": [
+          "autoscaling:SetDesiredCapacity",
+          "autoscaling:TerminateInstanceInAutoScalingGroup",
+          "ec2:DescribeImages",
+          "ec2:GetInstanceTypesFromInstanceRequirements",
+          "eks:DescribeNodegroup"
+        ],
+        "Resource": ["*"]
+      }
+    ]
+  }
+  ```
+<br />
+
+* **Updated Kubernetes layer manifest** - Kubernetes pack's manifest should be updated with the newly created IAM policy ARN. The YAML code block below shows the `managedMachinePool.roleAdditionalPolicies` section to update in the Kubernetes pack's manifest. Refer to the **Usage** section below for more details with an example. 
+<br />
+
+  ```yaml
+  managedMachinePool:
+    #roleName: {{ name of the self-managed role | format "${string}" }}
+    ## A list of additional policies to attach to the node group role
+    roleAdditionalPolicies:
+    - {{ arn for the policy1 | format "${string}" }}
+  ```
+
+
+## Usage
+
+The first step is to define an IAM policy in the AWS account associated with Palette. Then, switch to Palette to update the Kubernetes layer manifest and add the AWS Cluster Autoscaler pack. Finally, deploy your cluster and scale down the instance type to validate that AWS Cluster Autoscaler works correctly. The following sub-sections will provide more detailed instructions.

```

Comment:
@sudkul87, for all these steps, number them so that the user understands that they need to be followed sequentially. See the Ubuntu pack page for example: https://docs.spectrocloud.com/integrations/ubuntu